KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -18,7 +18,16 @@ default configuration = 'Debug'
 
 @{
     var projectFolder = Path.GetDirectoryName(projectFile);
```

Comment:
interesting. but if you run `dotnet test --help` you don't get the runner arguments